CanvasPlusPlayground/Common/Components/CustomizeCourseView.swift

```diff
@@ -0,0 +1,316 @@
+//
+//  CustomizeCourseView.swift
+//  CanvasPlusPlayground
+//
+//  Created by Ethan Fox on 9/17/25.
+//
+
+import Foundation
+import SwiftUI
+
+#if os(iOS)
+struct CustomizeCourseView: View {
+    let courseName: String
+    @State var selectedSymbol: String
+    @State var selectedColor: Color
+    
+    let symbolList: [String] = CourseCustomizationOptions.allowedSymbols
+    
+    let colorList: [Color] = CourseCustomizationOptions.allowedColors
+    
+    @Environment(\.dismiss) var dismiss
+    let onDismiss: (String, Color) -> Void
+    
+    
+    var body: some View {
+        NavigationStack {
+            Form {
+                headerCard
+                
+                Section {
+                    colorPicker
+                } header: {
+                    Text("Choose a Color")
+                }
+                
+                Section {
+                    iconPicker
+                } header: {
+                    Text("Choose an Icon")
+                }
+            }
```

Comment:
worth adding a title
```suggestion
            }
            .navigationTitle("Customize \(courseName)")
```

Reply:
I'm going to make it customize course because a long course name leads to poor truncation

<img width="1206" height="2622" alt="image" src="https://github.com/user-attachments/assets/590343db-80b3-4a21-b505-af252aaecf9e" />